src/main/java/duke/command/DeadlineCommand.java

```diff
@@ -0,0 +1,40 @@
+package duke.command;
+
+import duke.DukeException;
+import duke.storage.Storage;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.task.Deadline;
+import duke.ui.Ui;
+
+public class DeadlineCommand implements Command {
+
+    private final String INPUT; //deadline return book /by saturday
+
+    public DeadlineCommand(String input) {
+        this.INPUT = input;
+    }
+
+    @Override
+    public void execute(TaskList taskList, Ui ui, Storage storage) throws DukeException {
+        if (INPUT.isEmpty()) {
+            throw new DukeException("Exceed Charge....\n\t " +
```

Comment:
I like that it provides guidance to the user for each command when an invalid input is entered.